src/Backupper/SQLite/Backupper.php

```diff
@@ -82,4 +82,10 @@ private function getTablesToBackup(): array
 
         return $tables;
     }
+
+    #[\Override]
+    protected function getBuiltinDefaultOptions(): string
+    {
+        return '-bail';
```

Comment:
I don't even know what this is ^^

Reply:
The SQLite help gives this description: "stop after hitting an error".